upgrade from 5.2 to 5.3 steps

Jurj-Bogdan opened the pull request:
Let me know of any changes. Also, should it be more in depth?

closes #84

arhimede commented on docs/book/v5/upgrading/UPGRADE-5.3.md:
```diff
@@ -0,0 +1,159 @@
+  > Note that you can set the memory limit to a value of your choosing, with a recommended minimum of 256M
+
+### Update anonymization
+
```
1 pharse WHAT 
1 phrase WHY

arhimede commented:
@Jurj-Bogdan you need to edit another file to have that page added to menu 
docs/book/v5/upgrading/UPGRADE-5.3.md

bidi47 commented:
> @Jurj-Bogdan you need to edit another file to have that page added to menu docs/book/v5/upgrading/UPGRADE-5.3.md

it's this file that is used to place the links in the menu
https://github.com/dotkernel/api-documentation/blob/main/mkdocs.yml

Jurj-Bogdan commented:
@bidi47 @arhimede 

Added a few phrases where applicable, and tweaked the `mkdocs.yml` file so the new pages are added to the menu.
Also moved the old `upgrading.md` under the new `upgrading`

bidi47 commented on mkdocs.yml:
```diff
@@ -22,7 +22,9 @@ nav:
           - "FAQ": v5/installation/faq.md
-      - Upgrading: v5/upgrading.md
+      - Upgrading:
+          - "Upgrading": v5/upgrading/upgrading.md
```
change to 'Upgrade procedure'

Jurj-Bogdan replied:
I've only changed the commented line (26), i hope you didn't mean the "main menu text" as well (line 25)

bidi47 commented on docs/book/v5/upgrading/UPGRADE-5.3.md:
```diff
@@ -11,27 +11,34 @@ This document only covers upgrading from version 5.2.
 ### Update anonymization
 
+By default, Dotkernel API uses "soft delete" for it's `User` entities in order to preserve the database entries.
+
```
do we need this much spacing between these phrases? the other sections also have 3 separate paragraphs, but i think it's better to merge them as they are on a single topic

bidi47 commented on docs/book/v5/upgrading/UPGRADE-5.3.md:
```diff
@@ -56,6 +63,12 @@ $user->setIdentity($placeholder . $this->config['userAnonymizeAppend']) //...
+
 * [BC Break] Remove the `isDeleted` property from the `User` class, alongside all usages, as seen in the [pull request](https://github.com/dotkernel/api/pull/359/files)
 * Add a new "deleted" case to `UserStatusEnum`, which is to be used instead of the previous `isDeleted` property
 * Update the database and it's migrations to reflect the new structure
```
change "it's" to "its"
